contrib/linux/README.md

```diff
@@ -0,0 +1,5 @@
+# cloud-init configuration
```

Comment:
Why not just use https://github.com/containerd/containerd/tree/v1.7.0-rc.0/contrib/ansible ?
(Outdated and needs update though)